File: get_logos.js
const https = require('https');

const slugs = ['netflix', 'amazon', 'meta', 'slack', 'google', 'apple', 'spotify', 'microsoft', 'figma', 'airbnb', 'adobe', 'uber'];

function fetchSvg(slug) {
  return new Promise((resolve, reject) => {
    const url = `https://cdn.jsdelivr.net/npm/simple-icons@v11/icons/${slug}.svg`;
    https.get(url, (res) => {
      let data = '';
      res.on('data', (chunk) => { data += chunk; });
      res.on('end', () => {
        if (res.statusCode === 200) {
          const match = data.match(/d="([^"]+)"/);
          if (match) {
            resolve({ slug, path: match[1] });
          } else {
            reject(new Error(`No path found for ${slug}`));
          }
        } else {
          reject(new Error(`Status ${res.statusCode} for ${slug}`));
        }
      });
    }).on('error', (err) => reject(err));
  });
}

async function main() {
  for (const slug of slugs) {
    try {
      const result = await fetchSvg(slug);
      console.log(`SLUG:${result.slug} PATH:${result.path}`);
    } catch (err) {
      console.error(`ERROR:${slug} - ${err.message}`);
    }
  }
}

main();

File: get_logos.py
import urllib.request
import re

slugs = ['netflix', 'amazon', 'meta', 'slack', 'google', 'apple', 'spotify', 'microsoft', 'figma', 'airbnb', 'adobe', 'uber']
for slug in slugs:
    url = f"https://cdn.jsdelivr.net/npm/simple-icons@v11/icons/{slug}.svg"
    try:
        with urllib.request.urlopen(url) as response:
            svg = response.read().decode('utf-8')
            # Extract path d="..."
            match = re.search(r'd="([^"]+)"', svg)
            if match:
                print(f"slug: '{slug}', path: '{match.group(1)}'")
            else:
                print(f"slug: '{slug}' path not found in svg: {svg[:100]}")
    except Exception as e:
         print(f"error {slug}: {e}")
